### DISCUSSION TITLE
2019-10-05-%E5%88%9D%E5%AD%A6python%E6%9C%89%E5%93%AA%E4%BA%9B%E7%BC%96%E8%BE%91%E5%99%A8%E6%8E%A8%E8%8D%90/

### DISCUSSION THREAD

**Discussion by giscus** [Fusyong/Fusyong.github.io#5] [Announcements]
# 2019-10-05-%E5%88%9D%E5%AD%A6python%E6%9C%89%E5%93%AA%E4%BA%9B%E7%BC%96%E8%BE%91%E5%99%A8%E6%8E%A8%E8%8D%90/

https://blog.xiiigame.com/2019-10-05-%E5%88%9D%E5%AD%A6python%E6%9C%89%E5%93%AA%E4%BA%9B%E7%BC%96%E8%BE%91%E5%99%A8%E6%8E%A8%E8%8D%90/

**Comment by ecomter** [comment]
我感觉如果 Py 配合 .net 了话 ，Visual Studio 也挺好的
